documentation/sphinx/source/release-notes/release-notes-710.rst

```diff
@@ -2,6 +2,17 @@
 Release Notes
 #############
 
+7.1.37
+======
+* Same as 7.1.36 release with AVX enabled.
+
+7.1.36
+======
+* Released with AVX disabled.
+* Fixed Fix the go tuple tests for new go version `(PR #10745) <https://github.com/apple/foundationdb/pull/10745>`_
```

Comment:
Let's delete this line, since it's a test fix for new go version, not a real bug fix.

Reply:
Done.